python/beeai_framework/tools/code/sandbox.py

```diff
@@ -89,7 +89,7 @@ async def _run(
             raise SandboxToolExecuteError.ensure(err)
 
     @classmethod
-    async def from_source_code(cls, url: str, env: dict[str, Any], source_code: str) -> Self:
+    async def from_source_code(cls, /, url: str, source_code: str, env: dict[str, Any] | None = None) -> Self:
```

Comment:
This is a breaking change.

Reply:
Thank you, I forgot to include BREAKING CHANGE in my commit message, I would include it in final commit

Reply:
Also this change brings it in line with TS, `env` should have always been optional